lib/autoload/active_record/associations/preloader/manual_has_many_through.rb

```diff
@@ -0,0 +1,5 @@
+class ActiveRecord::Associations::Preloader
+  class ManualHasManyThrough < HasMany
```

Comment:
Great. :+1:  
By the way, it's really really really hard to write something like `ManualHasManyThrough`...